src/insert_sort.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import timeit
+
+
+best_case = [x for x in range(0, 1000)]
+worst_case = best_case[::-1]
+
+
+def insert_sort(l):
+    """Sort a list by inserting values in order after comparing each value."""
+    if len(l) <= 1:
+        return
+    else:
+        for idx in range(1, len(l)):
+            cur = l[idx]
+            spot = idx
+            while spot > 0 and l[spot - 1] > cur:
+                l[spot] = l[spot - 1]
+                spot = spot - 1
+            l[spot] = cur
+
+
+if __name__ == '__main__':
```

Comment:
What's weird is that this looks correct and your best case/ worse case look like they are what they should be, but I'm getting the same time performance for both cases. Are you getting the same?